server/server.go

```diff
@@ -430,7 +430,12 @@ func (s *Server) releaseContainerName(name string) {
 
 // Shutdown attempts to shut down the server's storage cleanly
 func (s *Server) Shutdown() error {
-	_, err := s.store.Shutdown(false)
+	_, err := os.Stat("/var/run/ocid.shutdown")
```

Comment:
This file would not exist on /var/run since this is on tmpfs.  We could move it to /var/lib/ocid and then it would preserve.